hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/sink/clustering/ClusteringPlanOperator.java

```diff
@@ -49,6 +51,7 @@
  */
 public class ClusteringPlanOperator extends AbstractStreamOperator<ClusteringPlanEvent>
     implements OneInputStreamOperator<Object, ClusteringPlanEvent> {
+  private static final Logger LOG = LoggerFactory.getLogger(ClusteringPlanOperator.class);
 
```

Comment:
Is it used anywhere?

Reply:
Yes, there are several places in these two classes that use LOG to output all kinds of log message, but the `LOG` used are from their superclass. `LOG` are added only to make it easier to locate logs and make the output logs clear.